libcaf_net/caf/policy/scribe.hpp

```diff
@@ -88,9 +90,9 @@ class scribe {
     if (write_buf_.empty())
       return false;
     auto len = write_buf_.size() - written_;
-    void* buf = write_buf_.data() + written_;
+    auto buf = write_buf_.data() + written_;
     CAF_LOG_TRACE(CAF_ARG(handle_.id) << CAF_ARG(len));
-    auto ret = net::write(handle_, buf, len);
+    auto ret = net::write(handle_, as_bytes(make_span(buf, len)));
```

Comment:
This will not work without `as_bytes` because the implicit conversion from `span<byte>` to `span<const byte>` is not possible.
compile error: "error: could not convert ‘caf::make_span(buf, len)’ from ‘span<caf::byte>’ to ‘span<const caf::byte>’"

Reply:
Yep, `span` missed one constructor overload. I got a patch ready (waiting for Jenkins to push to `master`).